biz.aQute.bndlib/src/aQute/bnd/osgi/AnnotationHeaders.java

```diff
@@ -419,6 +455,29 @@ private void handleAttributeOrDirective(Annotation a) {
 		public void method(MethodDef defined) {
 			lastMethodSeen = defined.getName();
 		}
+
+		private Object getOrDefault(String name) {
+			Object object = annotation.get(name);
+
+			if (object == null) {
+				try {
+					object = c.getDefaults().get(name);
+				} catch (Exception e) {
+					throw Exceptions.duck(e);
+				}
+			}
+
+			if ((object instanceof TypeRef) && Target.class.getName()
```

Comment:
I think we will need a doc page in the PR to cover all the features like this little bit of magic. I am not sure I like reusing `Target` for this. Perhaps we should define a new (annotation?) type for this purpose.

Reply:
yeah, this was just a lazy solution. On the other hand it's never referenced by the user, so it could be an internal marker type.

Reply:
Should I just use `Object` instead?

Reply:
I was looking for a common type that would not be accidentally mistaken for some other usable type. Object is sometimes used in real scenarios. Also, I didn't want to force a type dependency for this feature, more of a convention (since this whole feature is just about convention).

Reply:
Don't use Object. I think we need to define a specific type which can be used here and has no other meaning. Classes are cheap.

Reply:
So which jar are we gonna force users to depend on when they code up their annotations?

Reply:
That is a fair question. But I am not a fan of reusing a class for a different purpose. I think `Target` is very close to the meaning here. Maybe we use it to avoid a new compile dependency.